server/productImporter.js

```diff
@@ -133,9 +133,11 @@ ProductImporter.createTopLevelProduct = function (product) {
   prod.price.min = minPrice;
   prod.price.range = minPrice + '-' + maxPrice;
   if (baseProduct.metafields) {
-    let delimited = baseProduct.metafields.split(',');
+    let delimited = baseProduct.metafields.split('|');
     prod.metafields = [];
+    console.log('here is prod title', prod.title);
```

Comment:
Not sure if you meant to leave these in here or not. I'm fine either way, but if for debugging might want to clean up the messages a bit.